config/config.go

```diff
@@ -37,16 +39,103 @@ var (
 	ErrRemoteConfigEmptyName = errors.New("remote config: empty name")
 )
 
+// Options allows configuration of the config package
+type Options struct {
```

Comment:
This don't really need to be externally accessible.

```suggestion
type options struct {
```